backend/views/syson-diagram-actionflow-view/src/test/java/org/eclipse/syson/diagram/actionflow/view/ActionFlowViewDiagramDescriptionTests.java

```diff
@@ -177,4 +178,10 @@ public void eachAQLServicesIsPublicServiceMethod() {
                 .collect(Collectors.toSet());
         new AQLExpressionCallsExistingServicesChecker(this.diagramServices).checkAll(aqlExpressions);
     }
+
+    @Test
+    @DisplayName("Diagram contains an empty diagram node")
+    public void diagramHasOnEmptyDiagramNode() {
+        new DiagramDescriptionHasOneEmptyDiagramNodeChecker().check(this.diagramDescription);
```

Comment:
Added some tests to check that diagrams that should have an empty diagram node have one, and diagrams that shouldn't don't